### PR TITLE
Igor/znk exercise pager

### PR DESCRIPTION
**Pull request by igormas**

### CODE REVIEW

**Comment by assafshp** [comment]
please merge zinkerz/dev (not master) and jenkins will also work

File: dist/znk-infra-main.css [hunk 1]
```diff
@@ -275,7 +275,8 @@ $blueBG8: #75cbe8;
 /* full background image with retina */
```
Comment (assafshp) [inline dist/znk-infra-main.css]: do we have to include the css file ? for the consumers of infra

File: test/karma-unit.conf.js [hunk 2]
```diff
@@ -43,7 +47,7 @@ module.exports = function (config) {
             'test/mock/**/*.js',
 
             //tests
-            'test/spec/**/*.spec.js'
```
Comment (assafshp) [inline test/karma-unit.conf.js]: I think it shoud be returned not only for znk-exercise

File: src/components/znkExercise/pager/znkExercisePagerDrv.js [hunk 3]
```diff
@@ -1,127 +1,117 @@
...
+                            var pagerItemElement = angular.element(domElement.querySelectorAll('.pager-item')[index]);
+                            if(status){
+                                pagerItemElement.addClass('bookmark');
+                            }else{
+                                pagerItemElement.removeClass('bookmark');
+                            }
+                        }
+
+                        function setPagerItemAnswerClass(index,question){
+                            var pagerItemElement = angular.element(domElement.querySelectorAll('.pager-item')[index]);
```
Comment (assafshp) [inline src/components/znkExercise/pager/znkExercisePagerDrv.js]: Is it possible to cache this element and not to look for it everytime

File: src/components/znkExercise/pager/znkExercisePagerDrv.js [hunk 4]
```diff
@@ -1,127 +1,117 @@
...
+                                    });
+                                };
+                                //render is not invoked for the first time
+                                ngModelCtrl.$render();
+                            },false);
+                        }
+
+                        scope.$parent.$watch(attrs.questions, function pagerQuestionsArrWatcher(questionsArr) {
+                            if (questionsArr) {
+                                scope.questions = questionsArr;
```
Comment (assafshp) [inline src/components/znkExercise/pager/znkExercisePagerDrv.js]: every time we override the scope.questions, is it ok

File: dist/znk-infra-main.css (hunk 1 above)
Comment (igormas) [inline dist/znk-infra-main.css]: Seperate task should be created for adding the scss to the dist folder.

File: test/karma-unit.conf.js (hunk 2 above)
Comment (igormas) [inline test/karma-unit.conf.js]: Fixed

**Comment by assafshp** [comment]
conflict....

**Pull request closed without merging**